src/ca/verify/index.ts

```diff
@@ -33,4 +34,6 @@ export function verifySigningCertificate(
   if (options.ctlogOptions.disable === false) {
     verifySCTs(trustedChain, trustedRoot.ctlogs, options.ctlogOptions);
   }
+
+  verifySignerIdentity(trustedChain[0], options.signers.certificateIdentities);
```

Comment:
Verification of the signing certificate will now include:

* Verification that the certificate chains back to a trusted root certificate
* Verification of signed certificate timestamps (SCTs) embedded in the certificate
* Verification that the issuer/SAN of the certificate matches some expected value